PYME/recipes/traits.py

```diff
@@ -30,4 +31,20 @@ def __init__(self, *args, **kwargs):
     def validate(self, object, name, value):
         # Traitsui hangs up if a file doesn't validate correctly and doesn't allow selecting a replacement - disable validation for now :(
         # FIXME
-        return value
\ No newline at end of file
+        return value
+
+class IntFloat(BaseFloat):
+    default_value = 0.0
+    info_text = 'Frankenstein to deal with automatic introspection'
+
+    def set(self, obj, name, value):
+        setattr(self, name, value)
```

Comment:
is `setattr()` call  needed? it seems a little weird. Ditto for the whole `set()` method - surely we can just let the base class handle it?

Reply:
Yeah, it doesn't work without the `set` method. I tried it. And `traits` docs actually say you need to explicitly define it.